test/time_tracker/timers/pubsub_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns time-tracker.timers.pubsub-test
   (:require [clojure.test :refer :all]
             [clojure.core.async :refer [chan alt!! put!] :as async]
+            [clojure.string :as str]
```

Comment:
@jysandy @trycatcher I think in most of our codebases `clojure.string` is usually required as `s`. Don't do `(:refer-clojure :exclude [str])` as it just pushes the real solution to the problem to the next person who genuinely needs both namespaces.